src/arguments.cpp

```diff
@@ -46,78 +40,78 @@ static const Multiplier UNIVERSAL[] = {{'n', 1}, {'u', 1000}, {'m', 1000000}, {'
-//     sched              - group threads by scheduling policy
-//     cstack=MODE        - how to collect C stack frames in addition to Java stack
-//                          MODE is 'fp', 'dwarf', 'lbr', 'vm' or 'no'
-//     clock=SOURCE       - clock source for JFR timestamps: 'tsc' or 'monotonic'
-//     alluser            - include only user-mode events
-//     fdtransfer         - use fdtransfer to pass fds to the profiler
-//     target-cpu=CPU     - sample threads on a specific CPU (perf_events only, default: -1)
-//     record-cpu         - record which cpu a sample was taken on
-//     simple             - simple class names instead of FQN
-//     dot                - dotted class names
-//     norm               - normalize names of hidden classes / lambdas
-//     sig                - print method signatures
-//     ann                - annotate Java methods
-//     lib                - prepend library names
-//     mcache             - max age of jmethodID cache (default: 0 = disabled)
-//     include=PATTERN    - include stack traces containing PATTERN
-//     exclude=PATTERN    - exclude stack traces containing PATTERN
-//     begin=FUNCTION     - begin profiling when FUNCTION is executed
-//     end=FUNCTION       - end profiling when FUNCTION is executed
-//     nostop             - do not stop profiling outside --begin/--end window
-//     ttsp               - only time-to-safepoint profiling
-//     title=TITLE        - FlameGraph title
-//     minwidth=PCT       - FlameGraph minimum frame width in percent
-//     reverse            - generate stack-reversed FlameGraph / Call tree (defaults to icicle graph)
-//     inverted           - toggles the layout for reversed stacktraces from icicle to flamegraph
-//                          and for default stacktraces from flamegraph to icicle
+//     start                   - start profiling
+//     resume                  - start or resume profiling without resetting collected data
+//     stop                    - stop profiling
+//     dump                    - dump collected data without stopping profiling session
+//     check                   - check if the specified profiling event is available
+//     status                  - print profiling status (inactive / running for X seconds)
+//     meminfo                 - print profiler memory stats
+//     list                    - show the list of available profiling events
+//     version                 - display the agent version
+//     event=EVENT             - which event to trace (cpu, wall, cache-misses, etc.)
+//     alloc[=BYTES]           - profile allocations with BYTES interval
+//     live                    - build allocation profile from live objects only
+//     nativemem[=BYTES]       - profile native allocations with BYTES interval
+//     nofree                  - do not collect free calls in native allocation profiling
+//     trace=METHOD[+DURATION] - method to be traced with optional latency threshold
```

Comment:
Why did you choose `+`?
`:` would look more natural.
Is it because of [condition](https://github.com/async-profiler/async-profiler/blob/6453ccca4364f4d8e0655945e0edc98e9ea0924c/src/profiler.cpp#L1008) in `selectEngine`? It can be changed to something smarter.